### PR TITLE
Feature/ros noetic port

### PR DESCRIPTION
**Pull request by pmusau17**
# PR Details

Update to package.xml and docker built script. Fix is for dns issues when installing in a Ubuntu 20.04 virtual machine.

## Motivation and Context

Building on Noetic is an intermediate step towards ROS2 Port. 

## How Has This Been Tested?

Tested on Ubuntu 20.04 and in a virtual machine. 

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docker/build-image.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ echo "Building docker image for $IMAGE version: $COMPONENT_VERSION_STRING"
 echo "Final image name: $USERNAME/$IMAGE:$COMPONENT_VERSION_STRING"
 
 cd ..
-docker build --no-cache -t $USERNAME/$IMAGE:$COMPONENT_VERSION_STRING \
+docker build --network=host --no-cache -t $USERNAME/$IMAGE:$COMPONENT_VERSION_STRING \
```
Comment (mjeronimo) [inline docker/build-image.sh:70]: @pmusau17  Why was this change needed? Just wondering before I commit.

Comment (pmusau17) [reply docker/build-image.sh:70]: This change was related to a dns error in a Ubuntu 20.04 virtual machine. Similar to this issue: https://github.com/moby/moby/issues/41003. There are other potential fixes, but letting the image build with access to the host network resolves this issue as well.

Comment (mjeronimo) [reply docker/build-image.sh:70]: Ok, how about adding a comment to that effect then, anticipating a similar question when we go to integrate upstream.

Comment (mjeronimo) [reply docker/build-image.sh:70]: Also, waiting on a response from Michael M., who's checking with a colleague.

**Review by mjeronimo — APPROVED** [review]
